package.json

```diff
@@ -14,10 +14,11 @@
     "private": true
   },
   "scripts": {
-    "start": "EMBER_CLI_CORDOVA=0 ember server --port 4202 --live-reload-port 35731",
-    "build": "EMBER_CLI_CORDOVA=0 ember build",
-    "test": "EMBER_CLI_CORDOVA=0 ember test --server",
-    "test:ci": "EMBER_CLI_CORDOVA=0 ember-pretty-test"
+    "start": "cross-env EMBER_CLI_CORDOVA=0 ember server --port 4202 --live-reload-port 35731",
+    "start:staging": "cross-env EMBER_CLI_CORDOVA=0 STAGING=true ember server --environment production --port 4202 --live-reload-port 35731",
```

Comment:
@ambikavaisag , why there's a need of staging build command ?

Reply:
@abhinav098 I added that because she works on the staging environment, as a shortcut